app/css/views/service.css

```diff
@@ -0,0 +1,6 @@
+.service__edit-tags-link {
+  width: 3rem;
+  height: 3rem;
+  padding-left: 1rem;
+}
```

Comment:
We should really use flexbox on the header. When the title is too big or the window width too narrow, the tag icon is wrapped horribly. Instead, the title should be ellipsis'ed and the icon should still be completely visible with nice padding around.